core/internal/testutils/pgtest/txdb.go

```diff
@@ -54,6 +54,9 @@ func init() {
 	} else if v2 == "" {
 		dbURL = v1
 		which = "DATABASE_URL"
+	} else if v1 == v2 {
```

Comment:
wouldn't this be better by grouping it in the previous conditional as `else if v2 == "" || v1 == v2` since it wil execute the same logic anyways?

Reply:
Updated